CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- RedHat-certified version of the secrets-provider-for-k8s image [cyberark/secrets-provider-for-k8s#93](https://github.com/cyberark/secrets-provider-for-k8s/pull/93)
```

Comment:
please change `RedHat` to `Red Hat`

Reply:
also - this: https://github.com/cyberark/secrets-provider-for-k8s/pull/93/files#r430225230